AndrewTehan/hometask-3/rainfall_statistics.rb

```diff
@@ -0,0 +1,51 @@
+# frozen_string_literal: true
+
+# this class is used for calculate mean and variance values for town rainfall
+class RainfallStatistics
+  attr_reader :town, :data
```

Comment:
I'd move it to `private` scope in this class, cause these readers are not used out of class